generate_tests.js

```
@@ -74,8 +73,7 @@ for (const file of defs) {
   // Safely quote the test description using JSON.stringify to handle special characters
   content.push(`  it(${JSON.stringify('bash: ' + command)}, async function() {`);
   content.push(`    const stdin = ${JSON.stringify(stdin)};`);
-  content.push(`    const args = ${JSON.stringify(parts.slice(1))};`);
-  content.push('    const result = await klsh.' + name + '.main({ args, stdin, env: {} });');
+  content.push('    const result = await klsh.klsh.main({ stdin: ' + JSON.stringify(command) + ', input: stdin, env: {} });');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)